draft-ietf-tls-iana-registry-updates.md

```diff
@@ -217,52 +263,122 @@ TLS_ECDHE_PSK_WITH_CHACHA20_POLY1305_SHA256   | {0xCC,0xAC}
 TLS_DHE_PSK_WITH_CHACHA20_POLY1305_SHA256     | {0xCC,0xAD}
 ~~~
 
-- Add the following:
+Despite the following behavior being crazy, experience has shown that
+some customers use the IANA registry as checklist against which to
+measure an implemention's completeness and some implementers blindly
+implement cipher suites.  Therefore, IANA [SHALL add/has added] the
+following warning:
+
+    WARNING: Cryptographic algorithms will be broken or weakened over
+    time.  Blindly implementing cipher suites listed here is not
+    advised.  Implementers and users need to check that the
+    cryptographic algorithms listed continue to provide the expected
+    level of security.
 
-    WARNING: Cryptographic algorithms will be broken or weakened over time.  Blindly implementing cipher suites listed here is not advised.  Implementers and users need to check that the cryptographic algorithms listed continue to provide the expected level of security.
+IANA [SHALL add/has added] the following note to ensure that those that
+focus on IANA registries are aware that TLS1.3 {{I-D.ietf-tls-tls13}}
+uses the same registry but defines ciphers differently: 
 
-    Note(1): Although TLS 1.3 uses the same cipher suite space as previous versions of TLS, TLS 1.3 cipher suites are defined differently, only specifying the symmetric ciphers, and cannot be used for TLS 1.2. Similarly, TLS 1.2 and lower cipher suites cannot be used with TLS 1.3.
+    Note(1): Although TLS 1.3 uses the same cipher suite space as
+    previous versions of TLS, TLS 1.3 cipher suites are defined
+    differently, only specifying the symmetric ciphers, and cannot be
+    used for TLS 1.2.  Similarly, TLS 1.2 and lower cipher suite values
+    cannot be used with TLS 1.3.
 
-    Note(2): Cipher suites marked as "Yes" are those allocated via Standards Track RFCs.  Cipher suites marked as "No" are not; cipher suites marked "No" range from "good" to "bad" from a cryptographic standpoint.
+IANA [SHALL add/has added] the following notes to document the rules
+for populating the Recommended column:
 
-    Note(3): The designated expert {{RFC5226}} only ensures that the specification is publicly available.
+    Note(2): Cipher suites marked as "Yes" are those allocated via
+    Standards Track RFCs.  Cipher suites marked as "No" are not; cipher
+    suites marked "No" range from "good" to "bad" from a cryptographic
+    standpoint.
+
+    Note(3): The designated expert {{RFC8126}} only ensures that the
+    specification is publicly available.
```

Comment:
Formatting note : indenting like this is a problem because you get a code block. You want to use >, like I did in #27.

Reply:
Ah that's what you were doing.  I'll fix this up.